app/views/application/_footer.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="footer-basic">
+    <footer>
+        <ul>
+            San Francisco Bay Bird Observatory ● <a href="https://maps.google.com/?q=524+Valley+Way,+Milpitas,+CA+95036&amp;entry=gmail&amp;source=g"> 524 Valley Way, Milpitas, CA 95035 </a> ● 408-946-6548 ● <a href="mailto:outreach@sfbbo.org" target="_blank">outreach@sfbbo.org</a>
+        </ul>
+        <br>
+        <ul>
+            <li>Images from Wikipedia</li>
```

Comment:
A list with a single item isn't much of a list. I suggest replacing the br/ul/li with a p instead. (Or putting both the attribution items in their own list, instead of having a separate ul for each of them).